### PR TITLE
fix: rta not starting for cpe for ui5 v1.71.x

### PR DESCRIPTION
**Pull request by Jimmy-Joseph19**
Fix for CPE Fails to load rta for ui5 version 1.71.x due to the following import statements
[import workspaceConnector from '../flp/WorkspaceConnector';](https://github.com/SAP/open-ux-tools/blob/544b23d4896ba4751d44b886baebe13d4ef9dda3/packages/preview-middleware-client/src/cpe/init.ts#L18)

### CODE REVIEW

File: packages/preview-middleware-client/src/cpe/connector-service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * A Class of WorkspaceConnectorService
+ */
+export class WorkspaceConnectorService {
+    /**
+     * Initializes connector service.
+     *
+     * @param sendAction action sender function
+     */
+    public async init(sendAction: ActionSenderFunction): Promise<void> {
+        const version = sap.ui.version;
```
Comment (voicis) [inline packages/preview-middleware-client/src/cpe/connector-service.ts:14]: We should avoid using globals. Please use `sap/ui/VersionInfo` instead here.

**Review by voicis — CHANGES_REQUESTED** [review]

File: packages/preview-middleware-client/src/cpe/connector-service.ts (hunk 1 above)
Comment (tobiasqueck) [reply packages/preview-middleware-client/src/cpe/connector-service.ts:14]: Could you run `npx @ui5/linter --details` on the code? It should find such issues.

Comment (Jimmy-Joseph19) [reply packages/preview-middleware-client/src/cpe/connector-service.ts]: Thanks for this command its very helpful. Probably we need an issue. there are currently 29 issues in the package preview-middleware-client. 
The one introduced with this PR is been resolved.

**Review by voicis — DISMISSED** [review]
- Code changes looks good
- Changeset present
- Good test coverage
- Review comments addressed

File: .changeset/heavy-windows-dance.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
```
Comment (zdravko-georgiev) [inline .changeset/heavy-windows-dance.md:2]: when a change is made in `preview-middleware-client`, then also a changeset for the `preview-middleware` needs to be added as well 😉 Please add one

Comment (Jimmy-Joseph19) [reply .changeset/heavy-windows-dance.md:2]: Thanks. Updated :)

**Review by voicis — APPROVED** [review]

**Review by zdravko-georgiev — APPROVED** [review]
Code is well structuted and easy to understand
Chageset is OK
Excellent test coverage

File: .changeset/heavy-windows-dance.md (hunk 2 above)
Comment (zdravko-georgiev) [reply .changeset/heavy-windows-dance.md:2]: thx

**Pull request merged**